app/build.gradle.kts

```diff
@@ -225,6 +225,12 @@ dependencies {
     implementation(libs.openai.client)
     implementation(libs.ktor.client.okhttp)
 
+    // Nostr
+    implementation(libs.rust.nostr)
+
+    // Markdown
+    implementation("org.jetbrains:markdown:0.7.3")
```

Comment:
Should move this to the dependency catalog in `settings.gradle.kts`

Reply:
Fixed.